fw/http.h

```diff
@@ -299,7 +299,8 @@ typedef struct {
 	TfwConn		*conn;						\
 	void (*destructor)(void *msg);					\
 	TfwStr		crlf;						\
-	TfwStr		body;
+	TfwStr		body;						\
+	int		trailers_len;
```

Comment:
We use it only with responses, maybe we should move `trailers_len` to `tfw_http_resp_t`? `cut` also located there.

Reply:
Yes, I agree. However, `process_trailer_hdr()` gets `TfwHttpMsg*` as argument, but it is called for both req an resp, so it seems no way to distinguish them here?